platformio run breaks when using RF24Network library

I'm trying to use RF24Network library with [platformio](http://platformio.org/). During the build of my project, I get an error while building the examples (see details below).

It seems to me that the `exclude` section in [`library.json`](https://github.com/TMRh20/RF24Network/blob/master/library.json) should contain `examples` and `examples_RPi`.

I do not have much experience with platformio (or even with C code in general), so I might well be completely wrong and problem on my side...

```
avr-g++ -o .pioenvs/nanoatmega328/RF24Network_ID435/examples_RPi/helloworld_rx.o -c -fno-exceptions -fno-threadsafe-statics -g -Os -Wall -ffunction-sections -fdata-sections -MMD -mmcu=atmega328p -DF_CPU=16000000L -DARDUINO_ARCH_AVR -DARDUINO_AVR_NANO -DARDUINO=10605 -I.pioenvs/nanoatmega328/FrameworkArduino -I.pioenvs/nanoatmega328/FrameworkArduinoVariant -I.pioenvs/nanoatmega328/Adafruit_DHT_ID19 -I.pioenvs/nanoatmega328/SPI -I.pioenvs/nanoatmega328/RF24_ID433 -I.pioenvs/nanoatmega328/RF24Network_ID435 .pioenvs/nanoatmega328/RF24Network_ID435/examples_RPi/helloworld_rx.cpp
.pioenvs/nanoatmega328/RF24Network_ID435/examples_RPi/helloworld_rx.cpp:12:23: fatal error: RF24/RF24.h: No such file or directory
#include <RF24/RF24.h>
^
compilation terminated.
scons: *** [.pioenvs/nanoatmega328/RF24Network_ID435/examples_RPi/helloworld_rx.o] Error 1
```

--- a/library.json
+++ b/library.json
@@ -8,9 +8,11 @@
     "url": "https://github.com/TMRh20/RF24Network.git"
   },
   "dependencies": { "name": "RF24", "authors": "TMRh20", "frameworks": "arduino" },
-  "exclude": [ 
-    "RPi",
-    "tests"
+  "exclude": [
+    "examples/*",
+    "examples_RPi/*",
+    "RPi/*",
+    "tests/*"
   ],
   "frameworks": "arduino",
   "platforms": [

